app/routes/product-details.$productSlug/product-details.module.scss

```diff
@@ -6,19 +6,21 @@
 }
 
 .productName {
-    font: 34px / 1.2 'Marcellus', serif;
+    font: var(--heading3);
+    font-size: 34px;
 }
 
 .sku {
     margin-top: 8px;
     font-size: 14px;
     line-height: 1;
-    color: #767165;
+    color: var(--colorA4);
 }
 
 .price {
     margin-top: 16px;
-    font: 21px / 1.3 'Marcellus', serif;
+    font: var(--heading5);
+    font-size: 21px;
```

Comment:
I think we should not override font variables. What's the point of having theme then.
If there is no such variable choose the closest one or adjust/add variables according to the values we use. Exceptions may exist, but it seems to me there are too many of them.

Reply:
I agree and I don't like it as well. 
I'm against of choosing the closest one. IMO 1px in design matters.
Adjust/add variables - I would do this.

Reply:
1px matters when we have a design. But we deal with values from the complete website. Sometimes I saw font-size like this:
```css
--scaling-factor: min((100* 1cqw)), 9999px);
font-size: max(0.5px, 0.0195313 * (var(--scaling-factor) - var(--scrollbar-width))) !important;
```
I don't think we want this in out template so I took an approximate value.

Reply:
Right, when we know exact value. 
In the case you mentioned I use some dynamic values like `5vw` and see how it works on window resize. 
If there are a lot of places we use it, it would be good to have it in variable...

@alisey Do you agree to fit some variable values to the values we use in the project? 
Also If I see a font-size 1-2px different from variable font-size, I can try to fit it.

Reply:
> Do you agree to fit some variable values to the values we use in the project?

I didn't understand the question.

Reply:
@Letushev We discussed the problem with @alisey and decided to optimise  values and variables after the entire UI is ready.